hibernate6-jpa/src/main/java/io/micronaut/configuration/hibernate6/jpa/JpaConfiguration.java

```diff
@@ -60,7 +60,6 @@ public class JpaConfiguration {
     private List<String> mappingResources = new ArrayList<>();
     private EntityScanConfiguration entityScanConfiguration;
 
-    private boolean compileTimeHibernateProxies;
```

Comment:
Not supported in hibernate6, no way to set custom byte code provider so dropped this for now.